app/Widgets/Literacy/Views/literacy_widget.blade.php

```diff
@@ -0,0 +1,73 @@
+<div class="card m-3">
+    @if(!$current->exists())
+        <div class="card-body">
+            <h3 class="card-title">{{ ucwords(trans('literacy::widget.title')) }}</h3>
+            <p>{{ trans('literacy::widget.missing_data') }}</p>
+        </div>
+    @else
+        <canvas id="literacy-chart" class="card-img-top" height="400" styles="width: 100%; height: auto;"></canvas>
+        <div class="card-body">
+            <h3 class="card-title">{{ $current->year_reported }} {{ ucfirst(trans('literacy::widget.title')) }}</h3>
+            <dl class="row">
+                <dt class="col-sm-5">{{ ucfirst(trans('literacy::widget.total')) }}</dt>
+                <dd class="col-sm-7">{{ number_format($current->total) }}</dd>
+            </dl>
+        </div>
+    @endif
+    <div class="card-footer text-muted">
+        <p class="credit">{{ ucfirst(trans('literacy::widget.provided_by')) }} <a href="https://data.worldbank.org/indicator/IT.NET.BBND" target="_blank" rel="nofollow">{{ trans('literacy::widget.provider') }}</a></p>
```

Comment:
@LeviCostello Do you mind fixing the attribution URL?  It points to the broadband subscription data.